RInAppMessaging/Classes/RInAppMessaging.swift

```diff
@@ -43,10 +44,7 @@ import Foundation
     /// Optional delegate for advanced features
     @objc public static weak var delegate: RInAppMessagingDelegate? {
         didSet {
-            inAppQueue.async {
-                notifyIfModuleNotInitialized()
-                initializedModule?.delegate = delegate
-            }
+            initializedModule?.delegate = delegate
```

Comment:
Why was the queue wrapping removed?
Because you're just setting a property?

Reply:
It wasn't wrapped before. I think it should be safe (I can't remember why I wrapped that in my last PR 😞 )